app/src/main/java/uc/edu/ozturkkl/traveltracker/locationServiceStub/LocationStub.kt

```diff
@@ -3,5 +3,5 @@ package uc.edu.ozturkkl.traveltracker.locationServiceStub
 import androidx.lifecycle.MutableLiveData
 
 class LocationStub {
-    //test case we can use to sift through a mutable list of locations
+    /** test case we can use to sift through a mutable list of locations */
```

Comment:
JavaDoc belongs above a class or method.  Since this is not JavaDoc, leave it as a one line comment, // .  Even better, add logic to this class, or remove it entirely.  An empty/unused class is clutter,  which is technical debt.